core/server/api_container/server/service_network/default_service_network.go

```diff
@@ -679,43 +683,158 @@ func (network *DefaultServiceNetwork) RemoveService(
 	return serviceUuid, nil
 }
 
-// TODO we could switch this to be a bulk command; the backend would support it
-func (network *DefaultServiceNetwork) PauseService(ctx context.Context, serviceIdentifier string) error {
+func (network *DefaultServiceNetwork) StartService(
+	ctx context.Context,
+	serviceIdentifier string,
+) error {
+	serviceIdentifiers := []string{serviceIdentifier}
+	_, erroredUuids, err := network.StartServices(ctx, serviceIdentifiers)
+	if err != nil {
+		return stacktrace.Propagate(err, "An error occurred while starting services")
+	}
+
+	for serviceUuid, erroredUuid := range erroredUuids {
+		return stacktrace.Propagate(erroredUuid, "An error occurred while starting service '%v'", serviceUuid)
+	}
+
+	return nil
+}
+
+func (network *DefaultServiceNetwork) StartServices(
```

Comment:
Support is there to start more than one service but we are only exposing `plan.start_service`.